src/lib/Microsoft.Health.Common/Auth/Provider/AzureCredentialProvider.cs

```diff
@@ -0,0 +1,38 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System;
+using Azure.Identity;
+
+namespace Microsoft.Health.Common.Auth
+{
+    public class AzureCredentialProvider : IAzureCredentialProvider
```

Comment:
>AzureCredentialProvider [](start = 17, length = 23)

We already have an interface for IAuthService for use with FHIR auth.  Do you think there is an option to unify? Or do you think they need to be separate?

Reply:
Just to clarify, I am okay leveraging this work and removing the IAuthService pieces if that makes more sense.

---
In reply to: [569059401](https://github.com/microsoft/iomt-fhir/pull/87#discussion_r569059401) [](ancestors = 569059401)

Reply:
I'll try to unify these in the next PR that I do for the fhir server. I need to spend a bit more time understanding how auth with the fhir server works and how it differs. We should be able to unify once I understand though.